units/systemd-binfmt.service.in

```diff
@@ -18,7 +18,7 @@ After=proc-sys-fs-binfmt_misc.automount
 After=proc-sys-fs-binfmt_misc.mount
 After=local-fs.target
 Before=sysinit.target shutdown.target
-ConditionPathIsReadWrite=/proc/sys/
+ConditionPathIsReadWrite=/proc/sys/fs/binfmt_misc
```

Comment:
as mentioned, please use ConditionPathIsMountPoint=, which would be best I am sure